sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveShim.scala

```diff
@@ -456,6 +463,14 @@ private[client] class Shim_v0_12 extends Shim with Logging {
   def listFunctions(hive: Hive, db: String, pattern: String): Seq[String] = {
     Seq.empty[String]
   }
+
+  override def getDatabaseOwnerName(db: Database): String = Utils.getCurrentUserName()
```

Comment:
then shall we return "" here, according to https://github.com/apache/spark/pull/26080/files#r353034438 ?

Reply:
make sense.